### PR TITLE
CAT-2590 refactored PhiroColorBrickTest

### PR DESCRIPTION
**Pull request by batmanandrobin**
Refactored PhiroColorBrickTest from Robotium to Espresso

### CODE REVIEW

**Comment by 84n4n4** [comment]
please run espresso tests

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/PhiroColorBrickTest.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Category({Cat.CatrobatLanguage.class, Level.Smoke.class})
+	@Test
+	public void testPhiroLightSpinner() {
+		onBrickAtPosition(0).checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(1).checkShowsText(R.string.brick_phiro_rgb_led_action);
+
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/PhiroColorBrickTest.java]: do not check every spinner value manually by picking it and then looking if it is there.

		List<Integer> spinnerValues = new ArrayList<>();
		spinnerValues.add(R.string.phiro_light_both);
		spinnerValues.add(R.string.phiro_light_left);
		spinnerValues.add(R.string.phiro_light_right);

		onBrickAtPosition(1).onSpinner(R.id.someBrickSpinnerId)
				.checkValuesAvailable(spinnerValues);

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/PhiroColorBrickTest.java [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+		List<Integer> spinnerValues = new ArrayList<>();
+		spinnerValues.add(R.string.phiro_light_both);
+		spinnerValues.add(R.string.phiro_light_left);
+		spinnerValues.add(R.string.phiro_light_right);
+		onBrickAtPosition(1).onSpinner(R.id.brick_phiro_rgb_light_spinner).checkValuesAvailable(spinnerValues);
+
+		onData(instanceOf(PhiroRGBLightBrick.class))
```
Comment (84n4n4) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/PhiroColorBrickTest.java]: please use the brickDataInteractionWrapper....

and if youre clicking through all the available spinner values, dont, you already asserted above that they are available.

**Comment by 84n4n4** [comment]
please use the brickDataInteractionWrapper when accessing brick fields, its getting pretty unreadable when every single click is a paragraph of code...

since if already mentioned in the other PR, this of course applies to all open PRs.

also, once you asserted that a spinner contains all values, dont try clicking on every single one of them. its unnecessary and makes tests flaky, excessively long, and hard to read...

**Comment by batmanandrobin** [comment]
Optimized the test with the brickDataInteractionWrapper and created a parameterized test.
The test is now much shorter and easier to understand.

**Pull request merged**